docs/server/source/data-models/asset-model.md

```diff
@@ -18,3 +18,11 @@ In a `TRANSFER` transaction, the `"asset"` must contain exactly one key-value pa
     "id": "38100137cea87fb9bd751e2372abb2c73e7d5bcf39d940a5516a324d9c7fb88d"
 }
 ```
+
+
+.. note::
```

Comment:
This is a Markdown file so the `.. note::` thing isn't going to work. (That's a Sphinx reStructuredText thing.) I will make the edit since there are some other minor things I'd like to change.